packages/dashboard/src/root/navbar.js

```diff
@@ -166,7 +211,11 @@ class DashboardNavbar extends LitElement {
         <div class="tabs">
           <vaadin-tabs .selected=${this.selectedTabIndex} @selected-changed=${this.#onTabChange}>
             ${tabs.map(({ label }, index) => html`
-            <vaadin-tab value=${index}>${label}</vaadin-tab>
+            <vaadin-tab value=${index}>${label} 
+              <vaadin-button class="delete-tab-button" theme="icon tertiary" aria-label="Delete Tab" deletes=${label} @click=${()=>this.deleteTab(label)}
```

Comment:
I feel like the name logic in `onAddTab` and `deleteTab` function is overkill, and `() =>  removeElement(tabs[index], connector)` would probably work fine here. What do you think?

Reply:
It’s overkill if the ordering in #getTabs is guaranteed to be equal to the indices in the html. You’re referring to the search by name?